scss/login/_login.scss

```diff
@@ -182,9 +186,11 @@ div.delta-form-group {
       }
     }
 
-    .bp3-button.bp3-minimal.bp3-intent-warning,
-    .bp3-button.bp3-minimal.bp3-intent-warning:hover {
+    .bp3-button.bp3-minimal.bp3-intent-warning {
       color: #62656a !important;
+      &:hover {
+        color: white !important;
+      }
     }
```

Comment:
where is this used?

Reply:
It fixes the show password icon being all grey on hovering. Fixed it on the go.